packages/skin/src/routes/_index/component/chip/+page.marko

```diff
@@ -26,10 +26,366 @@
+            <button
+                class="chip chip--discovery"
+                type="button"
+                aria-pressed="false"
+            >
+                <img
+                    class="chip__media"
+                    src="https://ir.ebaystatic.com/cr/v/c01/skin/docs/dog_profile_optimized.jpg"
+                    alt=""
+                >
+                <span class="chip__text">
+                    Football
+                </span>
+            </button>
+            <span style="margin-right: 8px;"/>
+            <a class="chip chip--discovery" href="https://www.ebay.com">
+                <img
+                    class="chip__media"
+                    src="https://ir.ebaystatic.com/cr/v/c01/skin/docs/dog_profile_optimized.jpg"
+                    alt=""
+                >
+                <span class="chip__text">
+                    Football
+                </span>
+            </a>
+        </div>
+    </div>
+
+    <highlight-code type="html">
+        <button class="chip chip--discovery" type="button" aria-pressed="false">
+            <img
+                class="chip__media"
+                src="https://ir.ebaystatic.com/cr/v/c01/skin/docs/dog_profile_optimized.jpg"
+                alt=""
+            >
+            <span class="chip__text">
+                Football
+            </span>
+        </button>
+
+        <a class="chip chip--discovery" href="https://www.ebay.com">
+            <img
+                class="chip__media"
+                src="https://ir.ebaystatic.com/cr/v/c01/skin/docs/dog_profile_optimized.jpg"
+                alt=""
+            >
+            <span class="chip__text">
+                Football
+            </span>
+        </a>
+    </highlight-code>
+
+    <h3>Toggle discovery chip selected</h3>
+    <p>
+        For an link selected toggle chip, it should have a clipped selected text.
+    </p>
+    <div class="demo">
+        <div class="demo__inner">
+            <button
+                class="chip chip--discovery"
+                type="button"
+                aria-pressed="true"
+            >
+                <img
+                    class="chip__media"
+                    src="https://ir.ebaystatic.com/cr/v/c01/skin/docs/dog_profile_optimized.jpg"
+                    alt=""
+                >
+                <span class="chip__text">
+                    Football
+                </span>
+            </button>
+            <span style="margin-right: 8px;"/>
+            <a
+                class="chip chip--selected chip--discovery"
+                href="https://www.ebay.com"
+            >
+                <img
+                    class="chip__media"
+                    src="https://ir.ebaystatic.com/cr/v/c01/skin/docs/dog_profile_optimized.jpg"
+                    alt=""
+                >
+                <span class="chip__text">
+                    Football
+                    <span class="clipped">
+                        - selected
```

Comment:
For this text in all cases, lets go with " - filter applied". This is what we use on the SRP left rail filters. This also avoids any overlap with ARIA terminology (e.g. "selected")